shared/store/wallet.ts

```diff
@@ -84,10 +86,18 @@ export const fetchPoolPairs = createAsyncThunk(
 
 export const fetchTokens = createAsyncThunk(
   'wallet/fetchTokens',
-  async ({ size = 200, address, client }: { size?: number, address: string, client: WhaleApiClient }): Promise<{ tokens: AddressToken[], utxoBalance: string }> => {
+  async ({ size = 200, address, client }: { size?: number, address: string, client: WhaleApiClient }): Promise<{ tokens: WalletToken[], utxoBalance: string }> => {
     const tokens = await client.address.listToken(address, size)
+    if (!tokens.some((t) => t.id === '0')) {
+      tokens.push(tokenDFI)
+    }
+    const tokensWithPrice: WalletToken[] = await Promise.all(tokens.map(async (token) => {
+      const activePrices = await client.prices.getFeedActive(token.symbol, 'USD', 1)
```

Comment:
Are you getting prices from oracles? The main requirement for this ticket is to get the prices from DEX, not oracles.

Reply:
No need to fetch from API, or maybe once. But prices should be derived from DEX